org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -418,6 +426,58 @@ SsoAuditor:
     sessionDuration: 'PT1H'
     masterAccountId: !Ref MasterAccount
 
+SsoFinanceAuditor:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.7/templates/SSO/aws-sso.njk
+  TemplatingContext:
+    customerManagedPolicies:
+      - Name: !Ref CostExplorerPolicyName
+  StackName: !Sub '${resourcePrefix}-${appName}-finance-auditor'
+  StackDescription: 'Role used by Finance Auditor group within whole organization'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: '*'
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref financeAuditorGroup
+    permissionSetName: 'FinanceAuditor'
+    managedPolicies:
+      - 'arn:aws:iam::aws:policy/job-function/SupportUser'
+      - 'arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess'
+    sessionDuration: 'PT1H'
```

Comment:
1hr is pretty short, wondering if we should just bump this up slightly?  maybe 2-3 hrs?

Reply:
good point, I'm not sure why the original auditor role is so short, but it looks like most of the other roles are 8 hours, so I'll use that for the finance role (but keep the security role at 1 hour).